### PR TITLE
CT-371 typescriptify expressApp

### PR DESCRIPTION
**Pull request by dannybitgo**
https://bitgoinc.atlassian.net/browse/CT-371

### CODE REVIEW

**Comment by dannybitgo** [comment]
It didn't seem like there was too much to change here but lmk if you think I should try to do more

**Comment by dannybitgo** [comment]
> Good start, but I think we are missing some types, and it would also be good to move away from the `module.exports.blah` style and start using the ES6 syntax `export function blah`

How you export something that is a Coroutine, like this one:
```
module.exports.init = co(function *() {
```

File: modules/express/src/expressApp.ts [hunk 1]
```diff
@@ -282,13 +283,13 @@ module.exports.app = function(cfg: Config) {
 
 module.exports.init = co(function *() {
```
Comment (dannybitgo) [inline modules/express/src/expressApp.ts]: not sure how to change this to the Es6 format

File: modules/express/src/expressApp.ts [hunk 2]
```diff
@@ -168,7 +169,7 @@ function createHttpServer(app) {
  * @param baseUri
  * @return {Function}
  */
-module.exports.startup = function(config: Config, baseUri: string) {
+export function startup(config: Config, baseUri: string): any {
```
Comment (tylerlevine) [inline modules/express/src/expressApp.ts]: return type here is `() => void` (startup returns a function which returns void)

File: modules/express/src/expressApp.ts (hunk 1 above)
Comment (tylerlevine) [reply modules/express/src/expressApp.ts]: You can do this:
```
export function init() {
  return co(function*() {
    // you can use yield in here
  }).call(this);
}
```

**Pull request merged**